kyaml/yaml/alias.go

```diff
@@ -49,3 +56,15 @@ var FoldedStyle yaml.Style = yaml.FoldedStyle
 var LiteralStyle yaml.Style = yaml.LiteralStyle
 var SingleQuotedStyle yaml.Style = yaml.SingleQuotedStyle
 var TaggedStyle yaml.Style = yaml.TaggedStyle
+
+var enableSeqIndent bool
+func EnableSeqIndent() {
+	enableSeqIndent = true
+}
+
+var indent int
+var indentSet bool
+func SetIndent(i int) {
+	indentSet = true
+	indent = i
+}
```

Comment:
Suggestions: 
* Instead of a boolean tracking whether indent has been changed, how about a `defaultIndent` constant an initializing like `var indent = defaultIndent`?
* What's your opinion on accessors like this vs. exposing `Indent`, `DefaultIndent`  and `IndentSequences` directly? With the accessors approach, I think we'll need more, i.e. to reset the indent and disable sequence indentation.
* Nit: it's not really enabling/disabling indentation on sequences, so much as considering the sequence indicator part of the indentation. The distinction is meaningful for sequences that aren't top-level, or indentation sizes > 2. We could also refer to it a a sequence indentation style maybe? Naming this is hard for sure. 😅

Reply:
> Instead of a boolean tracking whether indent has been changed, how about a defaultIndent constant an initializing like var indent = defaultIndent?

Will do

> What's your opinion on accessors like this vs. exposing Indent, DefaultIndent and IndentSequences directly? With the accessors approach, I think we'll need more, i.e. to reset the indent and disable sequence indentation.

For now I left it as the accessor approach for the sake of error handling and added functions for resetting the indent and sequence indentation style. If you think it's better to directly expose the variables I'm okay with that too, I don't have a strong opinion - but library consumers might get weird error messages and/or panics.

> Nit: it's not really enabling/disabling indentation on sequences, so much as considering the sequence indicator part of the indentation. The distinction is meaningful for sequences that aren't top-level, or indentation sizes > 2. We could also refer to it a a sequence indentation style maybe? Naming this is hard for sure.

I tried to improve this by using better names and strings instead of bools - now someone can do something like `SetSequenceIndentationStyle(SequencesIndicatorIsNotPartOfIndentation)`. 

The constant names `SequenceIndicatorIsPartOfIndentation` and `SequencesIndicatorIsNotPartOfIndentation` are kind of long and verbose though, do you have any suggestions for how to make them more concise, or maybe a better way to make the distinction?

Reply:
I like the error handling you added. 👍 

I like the constants too. As for the specific names, maybe we could orient them around the visual effect. E.g. "compact" and "wide" or "spacey" 😆 ? `kyaml.SetSequenceIndentationStyle(kyaml.CompactSequenceStyle)`

Reply:
done